src/Symfony/Bundle/FrameworkBundle/Console/Descriptor/JsonDescriptor.php

```diff
@@ -204,6 +204,7 @@ protected function getRouteData(Route $route)
             'defaults' => $route->getDefaults(),
             'requirements' => $route->getRequirements() ?: 'NO CUSTOM',
             'options' => $route->getOptions(),
+            'condition' => '' !== $route->getCondition() ? $route->getCondition() : 'NO CONDITION',
```

Comment:
i see we use `NO CUSTOM` above already, but honestly it doesnt belong to a machine format. Here i'd use `null` instead of `NO CONDITION` or leave the field out (like XML does)

Reply:
Ok, i will leave the field out like XML